packages/kyt-cli/cli/actions/setup.js

```diff
@@ -539,12 +546,16 @@ module.exports = (flags, args) => {
     if (yarnOrNpm === 'yarn' && !args.packageManager) questions.push(ypmQ);
     ypm = args.packageManager ? args.packageManager : yarnOrNpm;
     if (!args.directory) questions.push(dirNameQ);
-    if (!args.repository) questions.push(skQ);
+    if (!args.repository && !localPath) questions.push(skQ);
 
     inquire.prompt(questions).then((answer) => {
       // question 1
       ypm = answer.ypm || ypm;
       // question 2
+      // Save Local directory Path before moving to new directory
+      if (localPath) {
+        localPath = path.join(process.cwd(), localPath);
```

Comment:
actually this answers my question above, this seems counterintuitive. why not just allow any path? if you join it to the cwd, then you can only use a relative path

```
> path.join(process.cwd(), '/usr/bin/')
'/<my_cwd>/usr/bin/'
```

how about using `path.resolve` instead?

```
> path.resolve('/usr/bin')
'/usr/bin'
> path.resolve('../usr/bin')
'/Users/usr/bin'
```